src/commands/console/org/select.js

```diff
@@ -45,18 +36,28 @@ class SelectCommand extends ConsoleCommand {
       aioConsoleLogger.debug(err)
       this.error(err.message)
     } finally {
-      cli.action.stop()
+      this.cleanOutput()
     }
   }
+
+  async selectOrgInteractive (preSelectedOrgIdOrCode) {
+    const orgs = await this.consoleCLI.getOrganizations()
+    const org = await this.consoleCLI.promptForSelectOrganization(
+      orgs,
+      { orgId: preSelectedOrgIdOrCode, orgCode: preSelectedOrgIdOrCode }
+    )
+    // Omit props
+    return { id: org.id, code: org.code, name: org.name }
+  }
 }
 
 SelectCommand.description = 'Select an Organization'
 
 SelectCommand.args = [
   {
     name: 'orgCode',
-    required: true,
-    description: 'Adobe I/O Org Code'
+    required: false,
```

Comment:
since this is not required anymore, would this be a breaking change

Reply:
good question, I don't think so because previous usage would still work

Reply:
ok LGTM